### DISCUSSION TITLE
Quick Transpose (Feature Request)

### DISCUSSION THREAD

**Discussion by Shoontz** [bangcorrupt/hacktribe#59] [Ideas]
On other Electribes you can quickly transpose a sequence in keyboard mode by holding shift and pressing a new key.
something like this on the Electribe 2 would open up ### real-time performance tremendously.

**Comment by bangcorrupt** [comment]
This is one of the first things I wanted to add.  I haven't hacked any of the menus or controls yet, so no idea how difficult it will be to implement.  There is a lot of interaction between different processes when a control is changed, it's difficult to hijack at a single point.  I'm going to make a lot of things configurable via SysEx before working out the UI.  

For now, you may be able to do read-modify-write on the pattern file via SysEx without interrupting the audio.  Would definitely need to use USB MIDI, TRS is too slow.

**Comment by Shoontz** [comment]
bangcorrupt, I could kiss you!  Thank you for all your hard work!

**Comment by ecco00** [comment]
man, just use the KEY menu item to transpose the one you're in... i mean, it's even available as a shortcut via shift+numpad :D

**Comment by Shoontz** [comment]
Hey ecco00, it's just not the same.  Having transpose work as it does on the older Electribes or the Beatstep would be such a killer real-time performance feature.  I think you'd probably have to use them to appreciate what a step backwards transpose is on the E2.

**Comment by ecco00** [comment]
can't see much difference, the only issue i see is the menu dive needed, but it could be mitigated with custom shortcuts (in the works, as i understand) via shift+padnum, do you agree?

**Comment by Shoontz** [comment]
You can see in this video how transpose used to work on the Electribes at about the 23-minute mark in this video.  :-)   https://youtu.be/O_3pnbTq26o?t=1436

**Comment by ecco00** [comment]
the true difference i see is the menu-dive needed
